src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -2014,7 +2014,7 @@ def process_declarations(self, parent, nodes, arg_list,
                 default_visibility = Symbol.Visibility.PUBLIC
```

Comment:
Can the `parent.symbol_table.default_visibility` be updated with this new info now?

If it can, maybe some more functions (that are below this point) can look at `parent.symbol_table.default_visibility` instead of passing it by argument.

Reply:
Oh yes, I've got rid of a few of those arguments now.